core/src/main/java/org/apache/calcite/adapter/enumerable/NullPolicy.java

```diff
@@ -32,8 +32,6 @@ public enum NullPolicy {
   STRICT,
   /** Returns null if one of the arguments is null, and possibly other times. */
   SEMI_STRICT,
-  /** If any of the arguments are null, return null. */
-  ANY,
```

Comment:
Why do you want to introduce a breaking change instead of deprecating it?

Reply:
This is mostly used by code generators so I guess usages outside calcite should be limited. Moreover, addressing the breaking change is rather straightforward so I don't think end-users will complain much. Entries were removed from this enum in the past and nobody complained about it.

If somebody feels strongly about deprecating first I am fine with it as well.